Utilities/DataFlow/include/DataFlow/HeartbeatSampler.h

```diff
@@ -31,14 +31,11 @@ class HeartbeatSampler : public Base::O2Device
 public:
   typedef AliceO2::Base::O2Message O2Message;
 
-  static constexpr const char* OptionKeyOutputChannelName = "output-channel-name";
+  static constexpr const char* OptionKeyOutputChannelName = "out-chan-name";
```

Comment:
Probably I forgot to comment on that last time, I thought I had spotted this. What's the reason for the renaming? For me `--output-channel-name` is the better keyword. But that's a minor detail.